scenarios/kubernetes_build.py

```diff
@@ -159,6 +161,8 @@ def main(args):
         '--extra-publish-file', help='Additional version file uploads to')
     PARSER.add_argument(
         '--allow-dup', action='store_true', help='Allow overwriting if the build exists on gcs')
+    PARSER.add_argument(
+        '--noupdatelatest', action='store_true', help='Do not update the latest file')
```

Comment:
Can we avoid the double negative here? `--skip-update` seems like a better name for this flag.

Reply:
+1

Reply:
If you insist, I will change it, however I do not see much point in changing the name of the flag that is just passed to underlying tool

Reply:
The tool should be fixed too (which is outside the scope of this PR)